ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -336,8 +350,15 @@ private void updateSubviewClipStatus(View subview) {
 
     // do fast check whether intersect state changed
     sHelperRect.set(subview.getLeft(), subview.getTop(), subview.getRight(), subview.getBottom());
-    boolean intersects = mClippingRect
-        .intersects(sHelperRect.left, sHelperRect.top, sHelperRect.right, sHelperRect.bottom);
+    Matrix matrix = subview.getMatrix();
+    if (!matrix.isIdentity()) {
```

Comment:
Got it working with https://github.com/facebook/react-native/pull/9456/files#diff-a73c2a0c22eceb34cf86aaede40f931fR192. Tested that it does indeed work by animating the transform prop in a clipped view and I can confirm that you were right and it did not work before but now works with this fix.